### PR TITLE
Fixes Blog, Pricing, Getting Started and Footer

### PR DESCRIPTION
**Pull request by Vinyl-Davyl**
Screenshot (143)
Screenshot (144)
Screenshot (145)
Screenshot (146)

### CODE REVIEW

**Review by taiwoAkindele — COMMENTED** [review]
Please attach a screenshot of the screen you worked on

**Comment by Vinyl-Davyl** [comment]
> Please attach a screenshot of the screen you worked on

Updated

File: src/components/home/Blog.tsx [hunk 1]
```diff
@@ -142,5 +143,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontSize: "18px",
     lineHeight: "22px",
     cursor: "pointer",
+    [theme.breakpoints.down("mobile")]: {
```
Comment (taiwoAkindele) [inline src/components/home/Blog.tsx:146]: I think tablet breakpoint should cover both tablet and mobile. Use mobile breakpoints only if you need to set a different value for mobile.

Comment (Vinyl-Davyl) [reply src/components/home/Blog.tsx:146]: Yeah
Thanks Baba, Just saw It
Would modify.

File: src/components/hub/ApiCard.styles.tsx [hunk 2]
```diff
@@ -17,6 +18,22 @@ export const useStyles = makeStyles((theme: Theme) => ({
     "&:hover": {
       boxShadow: " rgba(99, 99, 99, 0.2) 0px 0px 8px 0px",
     },
+
```
Comment (taiwoAkindele) [inline src/components/hub/ApiCard.styles.tsx:21]: Here also, the values in the laptop breakpoint and tablet are the same, remove the tablet and see how it goes

Comment (taiwoAkindele) [inline src/components/hub/ApiCard.styles.tsx:21]: Here also, the values in the laptop breakpoint and tablet are the same, remove the tablet and see how it goes

File: src/components/home/Blog.tsx (hunk 1 above)
Comment (Vinyl-Davyl) [reply src/components/home/Blog.tsx:146]: Modified

File: src/components/hub/ApiCard.styles.tsx (hunk 2 above)
Comment (Vinyl-Davyl) [reply src/components/hub/ApiCard.styles.tsx:21]: Sharp, It still work. Modified!

**Review by taiwoAkindele — APPROVED** [review]

**Pull request merged**